src/main/java/org/dependencytrack/resources/v1/BadgeResource.java

```diff
@@ -114,7 +114,73 @@ public Response getProjectVulnerabilitiesBadge(
                 if (project != null) {
                     final ProjectMetrics metrics = qm.getMostRecentProjectMetrics(project);
                     final Badger badger = new Badger();
-                    return Response.ok(badger.generate(metrics)).build();
+                    return Response.ok(badger.generateVulnerabilities(metrics)).build();
+                } else {
+                    return Response.status(Response.Status.NOT_FOUND).entity("The project could not be found.").build();
+                }
+            } else {
+                return Response.status(Response.Status.NO_CONTENT).build();
+            }
+        }
+    }
+
+    @GET
+    @Path("/violations/project/{uuid}")
+    @Produces(SVG_MEDIA_TYPE)
+    @ApiOperation(
+            value = "Returns current metrics for a specific project",
+            response = ProjectMetrics.class
```

Comment:
I know this is taken from the existing `/vulns/project/{uuid}` endpoint, but it's wrong. :D

Please change to a description that fits to what this endpoint actually does. Also change `response` to `String.class` or omit `response` entirely.

Reply:
Changed. I choose String.class because i found it in other parts of the code.